src/test/java/com/ericsson/ei/test/utils/TestConfigs.java

```diff
@@ -131,4 +125,11 @@ protected static void setupConnectionFactory(int port) throws IOException, Timeo
         connectionFactory.setConnectionTimeout(600000);
         connection = connectionFactory.newConnection();
     }
+
+    private static void setNewPortToMongoDBUriProperty(String mongoUri, String port) {
+        String modifiedUri = mongoUri.replaceAll("[0-9]{4,5}", port);
+        System.setProperty("spring.data.mongodb.uri", modifiedUri);
+        LOGGER.error("System property 'spring.data.mongodb.uri' changed from '{}' to '{}'",
```

Comment:
Why the error log?

Reply:
Was lazy to change the logging and I wanted to see it, should of course be debug, will fix.